spec/requests/providers/check_passported_answers_spec.rb

```diff
@@ -42,6 +42,7 @@
         expect(response.body).to include('Savings')
         expect(response.body).to include('assets')
         expect(response.body).to include('restrictions')
+        expect(response.body).not_to include(I18n.translate('.generic.none_declared'))
```

Comment:
This is a really important line of code. Without it the specs at 103, 110, and 117 have very limited value - in that this line tests the normal behaviour, and 103, 110, 117 contexts test how this behaviour is changed by modification to the application choices.

Therefore I would suggest this expectation is moved out to its own `it` clause, and that contexts 103, 110, 117 are moved higher within the file so that they immediately follow it.

With that change the specs will effectively read:

- This is the normal behaviour
- In this context x that behaviour changes
- In this context y that behaviour changes
- In this context z that behaviour changes